articles/service-fabric/service-fabric-get-started-linux.md

```diff
@@ -213,9 +213,8 @@ Service Fabric provides scaffolding tools that help you create Service Fabric ap
 
 After you install the generators, create guest executable or container services by running `yo azuresfguest` or `yo azuresfcontainer`, respectively.
 
-## Set up .NET Core 2.0 development
-
-Install the [.NET Core 2.0 SDK for Ubuntu](https://www.microsoft.com/net/core#linuxubuntu) to start [creating C# Service Fabric applications](service-fabric-create-your-first-linux-application-with-csharp.md). NuGet.org hosts packages for .NET Core 2.0 Service Fabric applications, currently in preview.
+## Set up .NET Core development
+Install the [.NET Core 3.1 SDK for Ubuntu](https://docs.microsoft.com/en-us/dotnet/core/install/linux-ubuntu) to start [creating C# Service Fabric applications](service-fabric-create-your-first-linux-application-with-csharp.md). NuGet.org hosts packages for .NET Core 3.1 Service Fabric applications.

```

Comment:
I dont think we changed the SDK dependency (on .NET Core 2.x) for our customers. Is there a specific reason we are changing this?

Reply:
Thinking more about this we should just say "Install the .NET Core SDK for Ubuntu" and remove the 2.0 bit. The original link (https://www.microsoft.com/net/core#linuxubuntu) already makes no mention to .net core 2.x and only references 3.1 which would be confusing to an end user.  We also don't need the tag on the link because it doesn't exist anymore.

Note we don't have a .net core 2.x dependency.  People simply need to consume a runtime that works on linux and can work with netstandard 2.0.

Reply:
I would suggest syncing with SDK team, who own this experience for SF .NET Core apps on Linux, before making the change.